src/rag/prompts.py

```diff
@@ -44,6 +44,32 @@
 ]
 """
 
+QUERY_GENERATOR_PROMPT_STRUCTURED_OUTPUT_TEMPLATE = """
```

Comment:
can you please update and use the existing `QUERY_GENERATOR_PROMPT_TEMPLATE` instead of creating a new one?

Reply:
done